src/plugins/appearence.js

```diff
@@ -0,0 +1,14 @@
+import prefix from '../prefix'
+
+export default {
+  noPrefill: ['appearance'],
+  supportedProperty: (prop) => {
+    if (prop === 'appearance') {
+      if (prefix.js === 'ms') {
```

Comment:
If we are on IE, we prefix with -webkit? maybe worth a comment

Reply:
Please take a look to - https://caniuse.com/#search=appearance
Everything ok. Edge browser should be prefixed with `-webkit-`

Reply:
I know, please add a comment "Microsoft Edge and IE Mobile support this property with the -webkit- prefix, rather than -ms- for interop reasons."